CRM/Civiquickbooks/Upgrader.php

```diff
@@ -138,4 +138,13 @@ public function upgrade_20203() {
 
     return true;
   }
+
```

Comment:
Is this upgrade function actually necessary?  I thought the Settings API returns the defaults for any setting that hasn't been added to the database...

Reply:
That could be. Not the easiest for me to test right now but I'm happy to remove this if it's not needed.